shared/teams/channel/create-channels.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react'
+import * as Kb from '../../common-adapters'
+import * as TeamsTypes from '../../constants/types/teams'
+import * as TeamsGen from '../../actions/teams-gen'
+import * as Container from '../../util/container'
+import * as Styles from '../../styles'
+import CreateChannels from '../new-team/wizard/create-channels'
+
+type Props = Container.RouteProps<{teamID: TeamsTypes.TeamID}>
+
+export default (props: Props) => {
+  const teamID = Container.getRouteProps(props, 'teamID', TeamsTypes.noTeamID)
+  const dispatch = Container.useDispatch()
+  React.useEffect(() => () => dispatch(TeamsGen.createSetChannelCreationError({error: ''})), [
+    props,
```

Comment:
I don't think you want to include props here, it includes stuff about the navigation state that you probably don't care to refresh on.